UPGRADE-4.1.md

```diff
@@ -104,3 +104,13 @@ Workflow
  * Deprecated the `add` method in favor of the `addWorkflow` method in `Workflow\Registry`.
  * Deprecated `SupportStrategyInterface` in favor of `WorkflowSupportStrategyInterface`.
  * Deprecated the class `ClassInstanceSupportStrategy` in favor of the class `InstanceOfSupportStrategy`.
+ * Deprecated passing the workflow name as 4th parameter of `Event` constructor in favor of the workflow itself.
+ * You can now define a place in XML (support more feature such as metadata) like following:
+
+   before:
+
+        <framework:place>first</framework:place>
+
+   after:
```

Comment:
do we need before/after? looks like a leftover of the previous deprecation, isn't it?